roles/monitoring_sap/tasks/hanadb-exporter-configuration.yml

```diff
@@ -0,0 +1,46 @@
+---
+# This playbook deploys prometheus-hanadb_exporter configuration on the monitoring host
+- name: Install prometheus-hanadb_exporter
```

Comment:
there is mixture of - and _. Keep it uniform

Reply:
this is the original package name. sadly.

Reply:
Cool